packages/loader/container-loader/src/container.ts

```diff
@@ -1420,7 +1420,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 }
             });
 
-        return protocol;
+        this._protocolHandler = protocol;
```

Comment:
we need to make sure this var get set in a synchronous context, or other things can happen after the object will be set is created, but not yet set

Reply:
made this a comment